src/cpp/redis.cpp

```diff
@@ -722,10 +722,28 @@ inline void Redis::_add_to_address_map(SRAddress& db_address)
 
 inline void Redis::_connect(SRAddress& db_address)
 {
+    // Build a connections object for this connection
+    // No need to repeat the build on each connection attempt
+    // so we do it outside the loop
+    sw::redis::ConnectionOptions connectOpts;
+    if (db_address._is_tcp) {
+        connectOpts.host = db_address._tcp_host;
+        std::cout << "Connecting to host: " + connectOpts.host + "; ";
```

Comment:
Can we move this into the logger somehow instead of outputting directly to STDOUT

Reply:
Debug text -- will remove